assignments/hackyourtemperature_W2/sources/keys.js

```diff
@@ -0,0 +1,9 @@
+// sources/keys.js
+import dotenv from "dotenv";
+dotenv.config(); // Загружает переменные из .env === loads variables from .env
+
+const keys = {
+  API_KEY: process.env.API_KEY || "" // fallback на пустую строку === fallback to empty string

```

Comment:
An empty string as a fallback value for API credentials can be confusing to people not familiar with the app. Consider throwing an error or log warning to make it more explicit what's happening.